pandas/io/sas/sas.pyx

```diff
@@ -437,7 +437,7 @@ cdef class Parser(object):
             elif column_types[j] == column_type_string:
                 # string
                 string_chunk[js, current_row] = np.array(source[start:(
-                    start + lngt)]).tostring().rstrip()
+                    start + lngt)]).tostring().rstrip(b"\x00 ")
```

Comment:
there is a space here as well?

Reply:
Hi @jreback 

Yes, there is supposed to be a space, just like many other places in sas7bdat.py. 

Apparently SAS can both pad with zeros and spaces to the right. (Maybe depending on version or something else).